### ISSUE TITLE
Regarding the pre-trained weights of the baseline model

### ISSUE THREAD

**Issue by apuomline** [RiqiangGao/GDP-HMM_AAPMChallenge#1]
Thank you for the baseline training code and the data visualization code you provided. They are very detailed. I would like to ask you, the pre-trained model you mentioned below:

![Image](https://github.com/user-attachments/assets/a5f1cf7e-750e-4ef6-92f0-16f553a77c82)
Was it trained using additional data, or was it trained using the training data from the current challenge?
I would also like to ask whether the MAE score of 2.54 for MedNext in the second stage used these pre-trained weights, or was it obtained by training from scratch using the baseline training code

**Comment by RiqiangGao** [comment]
Hi, @apuomline, thank you for your interest in the challenge! See answers below: 

1. only using the training data from the current challenge. 
You may notice that the train_lightning.py need the config file `config_files/config.yaml`, in config file the `'csv_root': 'meta_files/meta_data.csv'` is what we used for the baseline. `meta_data.csv` only contains data from this challenge. 

2. "pre-trained weights" and "training from scratch using the baseline training code" are basically the same thing. 
    The "pre-trained weights" is "training from scratch using the baseline training code" by me. However, you may get slightly different results because of random seed, hardware, or change of hyper-parameters.
